src/shogun/machine/StochasticGBMachine.cpp

```diff
@@ -401,11 +401,11 @@ void CStochasticGBMachine::init()
 	m_gamma=new CDynamicArray<float64_t>();
 	SG_REF(m_gamma);
 
-	SG_ADD((CSGObject**)&m_machine,"m_machine","machine");
-	SG_ADD((CSGObject**)&m_loss,"m_loss","loss function");
-	SG_ADD(&m_num_iter,"m_num_iter","number of iterations");
-	SG_ADD(&m_subset_frac,"m_subset_frac","subset fraction");
-	SG_ADD(&m_learning_rate,"m_learning_rate","learning rate");
-	SG_ADD((CSGObject**)&m_weak_learners,"m_weak_learners","array of weak learners");
-	SG_ADD((CSGObject**)&m_gamma,"m_gamma","array of learner weights");
+	SG_ADD(&m_machine, kMachine, "machine");
+	SG_ADD(&m_loss, kLoss, "loss function");
+	SG_ADD(&m_num_iter, kNumIterations, "number of iterations");
+	SG_ADD(&m_subset_frac, kSubsetFrac, "subset fraction");
+	SG_ADD(&m_learning_rate, kLearningRate, "learning rate");
+	SG_ADD(&m_weak_learners, kWeakLearners, "array of weak learners");
+	SG_ADD((CSGObject**)&m_gamma, kGamma, "array of learner weights");
```

Comment:
registering this as CDynamicArray<float64_t>* causes issues with cloning

Reply:
mmm that should be std::vector<float64_t> but yeah we'll take care of that laterz

Reply:
yea, just wasn't sure if everything was ready to start registering std::vector

Reply:
I think you can :)